src/config.js

```diff
@@ -483,6 +483,23 @@ const validators = {
     default: false,
     isClient: true
   }),
+  ONBOARDING_ASSIGNMENT_ID: num({
+    desc:
+      "Assignment ID used by api/lib/twilio-incoming to onboard new contacts",
+    default: undefined
+  }),
+  ONBOARDING_CAMPAIGN_ID: num({
+    desc: "Campaign ID used by api/lib/twilio-incoming to onboard new contacts",
+    default: undefined
+  }),
+  ONBOARDING_TEXT: str({
+    desc: "User ID used by api/lib/twilio-incoming to onboard new contacts",
+    default: undefined
+  }),
+  ONBOARDING_USER_ID: num({
```

Comment:
Would this be the userid of a blank dummy user created within spoke?

Reply:
Again, I am not totally clear on this.  The messages table has a user_id on it, as does the assignments table.  I assumed that this was the organization-level user.